<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GuiaINSSFolha</class>
 <widget class="QWidget" name="GuiaINSSFolha">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>609</width>
    <height>272</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="4" column="0" colspan="2">
    <widget class="QTableWidget" name="tableWidget">
     <property name="sortingEnabled">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QLineEdit" name="campoPesquisarObjetosTabela">
     <property name="clearButtonEnabled">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Período</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QDateEdit" name="periodoInicial">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="displayFormat">
         <string>MM/yyyy</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QDateEdit" name="periodoFinal">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="displayFormat">
         <string>MM/yyyy</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="QComboBox" name="tipoFolhaComboBox">
        <item>
         <property name="text">
          <string>11 - Folha Normal</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>12 - Folha Complementar</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>14 - Folha Complementar de Dissídio</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>31 - Folha Adiantamento de 13º</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>32 - Folha 13º Integral</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Filtros</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0" colspan="2">
       <widget class="QLineEdit" name="campoID_Empresa">
        <property name="placeholderText">
         <string>Código da Empresa</string>
        </property>
        <property name="clearButtonEnabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="QLineEdit" name="campoID_Filial">
        <property name="placeholderText">
         <string>Código da Filial</string>
        </property>
        <property name="clearButtonEnabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QPushButton" name="botaoProcessar">
        <property name="text">
         <string>Processar</string>
        </property>
        <property name="icon">
         <iconset resource="thema.qrc">
          <normaloff>:/images/paper-plane.png</normaloff>:/images/paper-plane.png</iconset>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QPushButton" name="botaoExportar">
        <property name="text">
         <string>Exportar</string>
        </property>
        <property name="icon">
         <iconset resource="thema.qrc">
          <normaloff>:/images/csv-file-format-extension.png</normaloff>:/images/csv-file-format-extension.png</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>periodoInicial</tabstop>
  <tabstop>periodoFinal</tabstop>
  <tabstop>tipoFolhaComboBox</tabstop>
  <tabstop>campoID_Empresa</tabstop>
  <tabstop>campoID_Filial</tabstop>
  <tabstop>botaoProcessar</tabstop>
  <tabstop>botaoExportar</tabstop>
  <tabstop>campoPesquisarObjetosTabela</tabstop>
  <tabstop>tableWidget</tabstop>
 </tabstops>
 <resources>
  <include location="thema.qrc"/>
 </resources>
 <connections/>
</ui>
